File: isma-intg-api/src/main/java/ru/nstu/isma/intg/api/IntgMetricData.java
package ru.nstu.isma.intg.api;

/**
 * @author Maria
 * @since 03.04.2016
 */
public class IntgMetricData {

    private long startTime;
    private long endTime;

    public void setStartTime(long startTime) {
        this.startTime = startTime;
    }

    public void setEndTime(long endTime) {
        this.endTime = endTime;
    }

    public long getSimulationTime() {
        return endTime - startTime;
    }
}

File: isma-solver/api/src/main/kotlin/ru/nstu/isma/intg/api/models/IntgMetricData.kt
package ru.nstu.isma.intg.api.models

/**
 * @author Maria
 * @since 03.04.2016
 */
class IntgMetricData {
    var startTime = 0L
    var endTime = 0L

    val simulationTime get() = endTime - startTime
}